src/main/Member.js

```diff
@@ -1,54 +1,75 @@
-import AuthHttpClient from "../http/AuthHttpClient";
-import HttpClient from "../http/HttpClient";
-import TransferTokenBuilder from "./TransferTokenBuilder";
-import Util from "../Util";
-import config from "../config.json";
-import AccessTokenBuilder from "./AccessTokenBuilder";
+// @flow
+import Util from '../Util';
+import config from '../config.json';
+import TokenRequest from './TokenRequest';
+import HttpClient from '../http/HttpClient';
+import AuthHttpClient from '../http/AuthHttpClient';
+import AccessTokenBuilder from './AccessTokenBuilder';
+import TransferTokenBuilder from './TransferTokenBuilder';
+import KeyStoreCryptoEngine from '../security/engines/KeyStoreCryptoEngine';
+import {
+    Key,
```

Comment:
Can we import `*` or are there conflicts?

Reply:
you have to `import * as ...` which means you have to destructure it anyway, so defeats the point.